FirebaseRemoteConfig/Tests/Unit/RCNRemoteConfigTest.m

```diff
@@ -1164,6 +1207,95 @@ - (void)testConfigureConfigWithValidInput {
   XCTAssertEqual([config valueForKey:@"_appName"], kFIRDefaultAppName);
 }
 
+#pragma mark app extension.
+- (void)testETagAndFetchTimeUpdatedInSharedUserDefaultsForAppExtension {
+  NSMutableArray<XCTestExpectation *> *expectations =
+      [[NSMutableArray alloc] initWithCapacity:RCNTestRCNumTotalInstances];
+  for (int i = 0; i < 1; i++) {
```

Comment:
This currently only runs for `i = 0`, is that expected? Or should it be `RCNTestRCNumTotalInstances`?